client/src/plus/search-filter/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function SearchFilter({
         <Button
           type="action"
           extraClasses={
-            selectedFilter === filter.param ? "active-menu-item" : undefined
+            selectedFilter === filter.param ? "active-menu-item" : null
```

Comment:
Wouldn't `null` be converted to `"null"` as well, if this was `extraClasses={"other-classes ${selectedFilter === filter.param ? "active-menu-item" : null}"}`, as `"" + null` gives `"null"`?

```suggestion
            selectedFilter === filter.param ? "active-menu-item" : ""
```

Reply:
I do not think so. The conditional with either return the string "active-menu-item" or `null`? As before, it would have returned "active-menu-item" or `undefined`. Either way, an empty string will have the same effect down the line.